mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/tilsagn/TilsagnService.kt

```diff
@@ -142,7 +142,7 @@ class TilsagnService(
 
             TilsagnStatus.TIL_FRIGJORING -> {
                 when (besluttelse.besluttelse) {
-                    Besluttelse.GODKJENT -> frigjorTilsagn(tilsagn, navIdent).right()
+                    Besluttelse.GODKJENT -> frigjorTilsagn(tilsagn, navIdent, sendMeldingTilOkonomi = true).right()
```

Comment:
Kan gjøre `okonomi.scheduleBehandleFrigjortTilsagn(tilsagn.id, session)` her i stedet for en if-sjekk i `frigjorTilsagn`?

Reply:
Ja